WordPress/Classes/Services/PostService+Revisions.swift

```diff
@@ -10,7 +10,7 @@ extension PostService {
     ///   - success: The success block accepts an optional list of Revisions
```

Comment:
If we go with the change proposed in this PR, we'd need to update this documentation comment:

```suggestion
    ///   - success: The success block
```

Alternatively, we could remove the whole docs for the parameters, since it merely restates what can be seen by looking at the type signatures.

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20267/commits/648f3703b02dcbb643ddc233e00d1bd25d56fe19